examples/consumer/index.js

```diff
@@ -3,13 +3,13 @@ const Fastify = require('fastify')
 const app = Fastify()
 
 app.register(require('fastify-amqp'), {
-  host: 'localhost'
+  hostname: 'localhost'
 }).after(function (err) {
```

Comment:
```suggestion
}).ready(function (err) {
```

Can you change? It has different behavior on fastify update.

Reply:
Actually, I'll adjust it in another PR.